npm/src/opentelemetry/metrics.ts

```diff
@@ -0,0 +1,35 @@
+import { metrics } from '@opentelemetry/api-metrics';
+
+const meter = metrics.getMeter('saml-jackson');
+
+const counters = {
+  createConfig: meter.createCounter('config_create', {
```

Comment:
`config_create` -> `saml.config.create`, etc. `.` is used to represent hierarchy, saml is the first grouping